src/extensions/default/Inline3DParametersEditor/InlineParameterEditor.js

```diff
@@ -0,0 +1,213 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var InlineWidget            = brackets.getModule("editor/InlineWidget").InlineWidget,
+        ParameterEditor         = require("ParameterEditor").ParameterEditor,
+        Inline3dParametersUtils = brackets.getModule("utils/Parameters3DUtils");
+
+    /** @type {number} Global var used to provide a unique ID for each parameter editor instance's _origin field. */
+    var lastOriginId = 1;
+
+    /**
+     * Inline widget containing a ParameterEditor control
+     * @param {!string} parameters  Initial Parameters
+     * @param {!CodeMirror.TextMarker} marker
+     */
+    function InlineParameterEditor(parameters, marker, tag) {
```

Comment:
This naming feels really generic to me.  Why did you avoid `Inline3DParameterEditor`?